opm/simulators/timestepping/AdaptiveTimeStepping.hpp

```diff
@@ -247,6 +247,10 @@ void registerAdaptiveParameters();
                     logException_(e, solverVerbose_);
                     // since linearIterations is < 0 this will restart the solver
                 }
+                catch (const ConvergenceMonitorFailure& e) {
+                    causeOfFailure = "Convergence monitor failure";
```

Comment:
All the others set the substepReport first, please flip order to avoid unnecessary differences.